app/gateways/http/authentication/middleware_test.go

```diff
@@ -0,0 +1,168 @@
+package authentication
+
+import (
+	"github.com/patriciapedrosaa/transfer-me/app/domain/authentication"
+	"github.com/patriciapedrosaa/transfer-me/app/domain/authentication/usecase"
+	"github.com/patriciapedrosaa/transfer-me/app/domain/entities"
+	http_server "github.com/patriciapedrosaa/transfer-me/app/gateways/http"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+	"time"
+)
+
+const (
+	authenticated = "authenticated"
+	idNotPresent  = "id not present"
+	notString     = "not string"
+	wrongReqId    = "wrong reqId"
+)
+
+func TestMiddleware(t *testing.T) {
```

Comment:
following the test naming convention, something like TestAuthenticate would be better, wouldn't it?